cg/services/order_validation_service/rules/case_sample/rules.py

```diff
@@ -209,16 +206,6 @@ def validate_volume_interval(order: OrderWithCases, **kwargs) -> list[InvalidVol
     return errors
 
 
-def validate_required_volume(order: OrderWithCases, **kwargs) -> list[VolumeRequiredError]:
```

Comment:
duplicated function, see line 202

Reply:
Line 202 is a different rule. If this one is broken the customers will see "Volume is missing" or something. If the other is broken, it will state that the volume is not within the allowed interval